### ISSUE TITLE
Review button is blinking when price is loading

### ISSUE THREAD

**Issue by elena-zh** [cowprotocol/cowswap#1668]
Reported in https://cowservices.slack.com/archives/C0361CDG8GP/p1670514950458299

When the nose picking cow is displayed, Limit order button becomes disabled . It makes an impression that the UI is always blinking. Can we not disable the button (like we do for the Swap form) when the cow is displayed?
![image.png](https://images.zenhubusercontent.com/5f55ec970a2e890ec10f7778/aec0ea21-3cda-44d7-b0ba-985c832c639f)

**Comment by shoom3301** [comment]
>Can we not disable the button (like we do for the Swap form) when the cow is displayed?

@elena-zh I don't think so. At the moment when we load a quote, we should disable trade button, because we don't know a price impact and can't display a warning for it.

**Comment by elena-zh** [comment]
Hey @shoom3301 , it would be great to fix it anyways. See the discussion: https://cowservices.slack.com/archives/C0361CDG8GP/p1671011757446689
